PatchKit/Classes/Extensions/UIViewController+Extensions.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  UIViewController+Extensions.swift
+//  SupportClient
+//
+//  Created by Omar Rasheed on 5/20/20.
+//  Copyright © 2020 Cornell Appdev. All rights reserved.
+//
+
+import UIKit
+
+extension UIViewController {
+
+    func removeBottomOfNavBar() {
+        navigationController?.navigationBar.layer.shadowOpacity = 0
+        navigationController?.navigationBar.layer.shadowColor = UIColor._mediumGray.cgColor
+        navigationController?.navigationBar.setBackgroundImage(UIImage(), for: UIBarMetrics.default)
+        navigationController?.navigationBar.shadowImage = UIImage()
+    }
+
+    func addBottomOfNavBar() {
+        navigationController?.navigationBar.layer.shadowOpacity = 0.5
+        navigationController?.navigationBar.layer.shadowColor = UIColor._mediumGray.cgColor
+        navigationController?.navigationBar.setBackgroundImage(nil, for: UIBarMetrics.default)
```

Comment:
porque this one is `nil` and the background image of the `removeBottomOfNavBar` is ``UIImage()`?

Reply:
not rly sure i just pulled this from the `IDK` project lmao also found this on stackoverflow. Ill mess around w it n see we need to be touching backgroundImage

Reply:
Oh so seems like the nav bar has some sort of `VisualEffectsView` and this just takes care of that. Gives the bar like an offwhite color if its not set to nil